test/conftest.py

```diff
@@ -20,123 +20,155 @@
 from sagemaker import LocalSession, Session
 from sagemaker.mxnet import MXNet
 
-from test.integration import NO_P2_REGIONS, NO_P3_REGIONS
+from utils import image_utils
 
 logger = logging.getLogger(__name__)
-logging.getLogger("boto").setLevel(logging.INFO)
-logging.getLogger("botocore").setLevel(logging.INFO)
-logging.getLogger("factory.py").setLevel(logging.INFO)
-logging.getLogger("auth.py").setLevel(logging.INFO)
-logging.getLogger("connectionpool.py").setLevel(logging.INFO)
+logging.getLogger('boto').setLevel(logging.INFO)
```

Comment:
if we use black to format this repo too, we're going to end up changing these back to double quotes 🤷‍♀️

Reply:
Git weird - I moved this file and didn't really change that.

Unfortunately we use both single  and double quotes all over the repo :(